content/docs/jsx-in-depth.md

```diff
@@ -308,7 +308,19 @@ You can mix together different types of children, so you can use string literals
 </div>
 ```
 
-A React component can't return multiple React elements, but a single JSX expression can have multiple children, so if you want a component to render multiple things you can wrap it in a `div` like this.
+A React component can return multiple elements:
```

Comment:
How do you feel about adding the word "also" like so?
> A React component can also return an array of elements:

Reply:
Sounds good to me! I'll update